app/src/main/java/org/simple/clinic/sync/indicator/SyncIndicatorState.kt

```diff
@@ -0,0 +1,8 @@
+package org.simple.clinic.sync.indicator
```

Comment:
Unrelated to this PR, but I was looking at `DataSync` and I think we should consider using a default value for `syncProgress` with a `BehaviorSubject` instead of `Publish` because it looks like you're already changing the value to `SUCCESS` when sync completes. 

If you find using `SUCCESS` uncomfortable then we could rename it to `IDLE`.

Reply:
Actually the default value for sync in terms of the indicator is not `SUCCESS`. It is `PENDING`.
Also, `DataSync` only comes into picture when we are actually syncing and not when we are scheduling syncs. So `SyncProgress` should not really have a default state. It should only be used to denote sync status when syncing starts.